src/lib/containers/evaluation_queues/EvaluationCard.tsx

```diff
@@ -34,95 +41,75 @@ export type EvaluationCardProps = {
  * This component is currently only intended to be used in Synapse.org.
  * For this reason, the dropdown menu items are unimplemented as no components
  * in this project implement their behavior. The dropdown options are also
- * disabled if the current user does not have permissions for the action
+ * not shown if the current user does not have permissions for the action
+ *
+ * All Evaluation metadata must be provided to this component; it will not
+ * retrieve an Evaluation via a REST API call
  */
 export const EvaluationCard: React.FunctionComponent<EvaluationCardProps> = ({
-  evaluationId,
+  evaluation,
   sessionToken,
   utc,
   onEdit,
   onModifyAccess,
-  onDelete,
   onSubmit,
 }: EvaluationCardProps) => {
   const [error, setError] = useState<SynapseClientError>()
-  const [evaluation, setEvaluation] = useState<Evaluation>()
   const [permissions, setPermissions] = useState<UserEvaluationPermissions>()
 
   useEffect(() => {
     //clear error
     setError(undefined)
 
-    getEvaluationPermissions(evaluationId!, sessionToken)
+    getEvaluationPermissions(evaluation.id, sessionToken)
       .then(retrievedPermissions => {
-        //clear error
         setPermissions(retrievedPermissions)
       })
       .catch(error => setError(error))
+  }, [evaluation, sessionToken])
 
-    getEvaluation(evaluationId!, sessionToken)
-      .then(retrievedEvaluation => {
-        setEvaluation(retrievedEvaluation)
-      })
-      .catch(error => setError(error))
-  }, [evaluationId, sessionToken])
+  const onDelete = () => {
+    setError(undefined)
+    deleteEvaluation(evaluation.id, sessionToken)
+      .then(() => {})
+      .catch(setError)
```

Comment:
Does this work?  I was expecting something like
.catch(error => setError(error))

Reply:
It does. `setError` is also a function with typing of: `(error) => void`